docs/tutorials/projects/send-security-photo.md

```diff
@@ -0,0 +1,313 @@
+
+```python
+yag.send('phone_number@gatewayaddress.com', 'subject', contents)
+```
+
+You will need to access to your phone number through your carrier.
+For this tutorial, you are going to send the text to yourself.
+You will replace `to@someone.com` with your phone number and [SMS gateway address](https://en.wikipedia.org/wiki/SMS_gateway).
+You can find yours here: [Gateway Addresses for Mobile Phone Carrier Text Message](https://support-en.wd.com/app/answers/detailweb/a_id/44431/~/gateway-addresses-for-mobile-phone-carrier-text-message).
+Some common ones:
+
+- AT&T: `txt.att.net`
+- T-Mobile:`tmomail.net`
+- Verizon Wireless: `vtext.com`
+
+As an example, if you have T-Mobile your code will look like this:
+
+```python
+yag.send('xxxxxxxxxx@tmomail.net', 'subject', contents)
+
+```
+
+This allows us to route the email to our phone as a text message.
+
+## Use the Viam Python SDK to control your security robot
+
+If you followed the [Use Object Detection to Turn Your Lights On](/tutorials/projects/light-up/) tutorial, you already set up a folder with some Python code that connects to your robot and implements the Vision Service.
+
+If you are starting with this tutorial, follow the [steps here](/tutorials/projects/light-up/#write-python-code-to-control-your-object-detection-robot) to create the main script file, connect the code to the robot, and select the model and label path.
+Ignore the step about the Kasa smart plug host address.
+
+Instead of using this person detector to activate a smart plug, you will send yourself a text message.
+
+Make a copy of the [<file>lightupbot.py</file>](https://github.com/viam-labs/devrel-demos/blob/main/Light%20up%20bot/lightupbot.py) file in your project directory and save it as <file>chocolate_security.py</file>.
+You will use the same robot connection code and Vision Service configuration code but edit some other parts of the file.
+
+Delete the `from kasa import Discover, SmartPlug` line and replace it with the following to import the Yagmail Python library:
+
+```python
+import yagmail
+```
+
+Now you need to rewrite the if/else function.
+If a person is detected, your robot will print `sending a message`, take a photo, and save it to your computer as <file>foundyou.png</file> (or whatever name you want).
+
+Then you will create a `yagmail.SMTP` instance to initialize the server connection.
+
+Refer to the code below and the [yagmail instructions](#how-to-use-yagmail) to edit your <file>chocolate_security.py</file> file as necessary.
+
+{{% expand "Click to show the full example code." %}}
+
+```python {class="line-numbers linkable-line-numbers" data-line="6"}
+import asyncio
+import os
+
+from viam.robot.client import RobotClient
+from viam.rpc.dial import Credentials, DialOptions
+from viam.components.camera import Camera
+from viam.components.types import CameraMimeType
+from viam.services.vision import VisionServiceClient, VisModelConfig, VisModelType, Detection
+from PIL import ImageDraw

```

Comment:
"from PIL import ImageDraw" not sure what this is

Reply:
i think in the first tutorial it may be nice to see this just to "learn" how it works, but in the second one we can make the code shorter by taking the frame section out as well as this line